mscore/musescore.cpp

```diff
@@ -406,6 +406,7 @@ void updateExternalValuesFromPreferences() {
       MScore::panPlayback = preferences.getBool(PREF_APP_PLAYBACK_PANPLAYBACK);
       MScore::playRepeats = preferences.getBool(PREF_APP_PLAYBACK_PLAYREPEATS);
       MScore::warnPitchRange = preferences.getBool(PREF_SCORE_NOTE_WARNPITCHRANGE);
+      MScore::pedalEventsMinTicks = preferences.getInt(PREF_IO_MIDI_PEDAL_EVENTS_MIN_TICKS);
```

Comment:
Do we really need to assign a `MScore` static variable for this preference? Since this preference is only used in `rendermidi.cpp`, why not directly refer to it as `preferences.getInt(PREF_IO_MIDI_PEDAL_EVENTS_MIN_TICKS)` there?

Reply:
That was my first intention, but then I realized that there is not any class from **libmscore** that directly gets any preference value that way. After all, correct me if I am wrong, but **mscore** depends on **libmscore**, and using `preferences.getInt()` would be like going in the wrong direction.

So, I understood that making that assignment to those static variables is the way (among others, I guess) the application **mscore** can configure some behaviors of library **libmscore**. 

Furthermore, I considered to add an extra parameter to `Score::renderMidi()`, but that option seemed to me more aggressive, since it looked a too specific option to be there. I also considered the possibility of add a new property or similar to `SynthesizerState` class, but I was not sure if it would have sense to put it there.

Reply:
Right, yes, sorry, didn't think of that.